### ISSUE TITLE
Misleading error message: first path segment in URL cannot contain colon

### ISSUE THREAD

**Issue by WyseNynja** [0xProject/0x-mesh#288]
I currently run my geth node behind haproxy for SSL and load balancing.

If you try to connect to the node outside my network, it requires HTTP basic authentication. However, if you are inside my network, the connection is allowed without auth. I have no coins on this node, so I think this is secure enough for this purpose.

When I try to start 0x-mesh with `ETHEREUM_RPC_URL="https://eth.stytt.com"`, I get this error:
```
{"error_string":"parse \"https://eth.stytt.com\": first path segment in URL cannot contain colon","level":"fatal","msg":"could not initialize app","time":"2019-07-21T17:21:13Z"}
```

When I try to start 0x-mesh with `ETHEREUM_RPC_URL=eth.stytt.com`, it reads the variable as a path to an IPC socket:
```
zrx_mesh_1  | {"config_core_Config":{"Verbosity":5,"DataDir":"0x_mesh","P2PListenPort":60558,"EthereumNetworkID":1,"UseBootstrapList":true,"OrderExpirationBuffer":10000000000,"BlockPollingInterval":5000000000,"EthereumRPCMaxContentLength":524288},"level":"info","msg":"Initializing new core.App","time":"2019-07-21T16:43:52Z","version_string":"1.0.6-beta"}
zrx_mesh_1  | {"error_string":"dial unix eth.stytt.com: connect: no such file or directory","level":"fatal","msg":"could not initialize app","time":"2019-07-21T16:43:52Z"}
```

What should I be putting for ETHEREUM_RPC_URL? It doesn't allow `"wss://eth.stytt.com"` or `"https://eth.stytt.com"` either.

**Comment by WyseNynja** [comment]
Aha! It didn't like the quotes. Removing the quotes fixed it. Apparently they work fine in the `docker run` command because bash removes them, but if you include them in a docker-compose.yml, they get passed through.

**Comment by WyseNynja** [comment]
Maybe this should be re-opened to fix the error message. "first path segment in URL cannot contain colon" is misleading.

**Comment by albrow** [comment]
Should be fixed by #306

**Comment by WyseNynja** [comment]
Thanks!
